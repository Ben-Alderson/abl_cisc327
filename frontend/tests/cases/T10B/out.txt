error(You must have account start with number other than 0)


