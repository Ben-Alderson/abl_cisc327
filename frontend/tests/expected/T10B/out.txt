Enter command: Enter login: Enter command: Enter the new account's number: Enter the new account's name: ERROR: Account number must be 7 numeric digits
Enter command: Enter command: 